src/transformers/models/auto/tokenization_auto.py

```diff
@@ -43,153 +43,157 @@
 
 logger = logging.get_logger(__name__)
 
-
-TOKENIZER_MAPPING_NAMES = OrderedDict(
-    [
-        ("retribert", ("RetriBertTokenizer", "RetriBertTokenizerFast" if is_tokenizers_available() else None)),
-        ("roformer", ("RoFormerTokenizer", "RoFormerTokenizerFast" if is_tokenizers_available() else None)),
-        (
-            "t5",
+if TYPE_CHECKING:
```

Comment:
`TYPE_CHECKING` needs to be imported to fix all the test errors.

Reply:
oops. thanks!